routes.js

```diff
@@ -1,17 +1,17 @@
-var Router = require('koa-router');
-var router = Router();
-var DB = require('./database');
-var db = DB();
+const Router = require('koa-router');
+let router = Router();
+const { Coffeegram } = require('./database');
 
-var pug = require('pug');
+const pug = require('pug');
 
 var home = pug.compileFile('templates/home.pug');
 var about = pug.compileFile('templates/about.pug');
 var upload = pug.compileFile('templates/new.pug');
 
 router.get('/', async ctx => {
+  var coffeegrams = await Coffeegram.find();
   ctx.body = home({
-    data: db
+    coffeegrams: coffeegrams
```

Comment:
You can use destructuring shorthand here: `coffeegrams: coffeegrams` => `coffeegrams`